[2024-07-23 15:30:20,091] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: ETL_task.load_from_mysql 2024-07-23T15:27:52.941443+00:00 [queued]>
[2024-07-23 15:30:20,103] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: ETL_task.load_from_mysql 2024-07-23T15:27:52.941443+00:00 [queued]>
[2024-07-23 15:30:20,104] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-07-23 15:30:20,104] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2024-07-23 15:30:20,104] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-07-23 15:30:20,112] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): load_from_mysql> on 2024-07-23T15:27:52.941443+00:00
[2024-07-23 15:30:20,119] {standard_task_runner.py:52} INFO - Started process 3691 to run task
[2024-07-23 15:30:20,124] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'ETL_task', 'load_from_mysql', '2024-07-23T15:27:52.941443+00:00', '--job-id', '339', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/ETL.py', '--cfg-path', '/tmp/tmp5_kmqugo', '--error-file', '/tmp/tmp8k6_7ta0']
[2024-07-23 15:30:20,126] {standard_task_runner.py:77} INFO - Job 339: Subtask load_from_mysql
[2024-07-23 15:30:20,169] {logging_mixin.py:104} INFO - Running <TaskInstance: ETL_task.load_from_mysql 2024-07-23T15:27:52.941443+00:00 [running]> on host 1026daea27fc
[2024-07-23 15:30:20,211] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=trihx2003@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=ETL_task
AIRFLOW_CTX_TASK_ID=load_from_mysql
AIRFLOW_CTX_EXECUTION_DATE=2024-07-23T15:27:52.941443+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-07-23T15:27:52.941443+00:00
[2024-07-23 15:30:20,267] {python.py:151} INFO - Done. Returned value was: True
[2024-07-23 15:30:20,291] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=ETL_task, task_id=load_from_mysql, execution_date=20240723T152752, start_date=20240723T153020, end_date=20240723T153020
[2024-07-23 15:30:20,315] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-23 15:30:20,337] {local_task_job.py:149} INFO - Task exited with return code 0
